src/bundle-manifest.ts

```diff
@@ -0,0 +1,160 @@
+/**
+ * A bundle is a grouping of files which serve the need of one or more
+ * entrypoint files.
+ */
+export interface IBundle {
```

Comment:
QQ: would you prefer maintaining an interface Bundle and then use simple objects without constructors to pass the data around or just use a Bundle class?